contracts/BridgeVault.sol

```diff
@@ -50,7 +49,8 @@ contract BridgeVault is Ownable, IBridgeVault {
         wETH.withdraw(amount);
 
         // Transfer the unwrapped ETH to the target address
-        targetAddress.transfer(amount);
+        // Send ether and specify a custom gas amount
+        targetAddress.call{value: amount/**, gas: 5000*/}("");
```

Comment:
Why this change?

Reply:
https://consensys.github.io/smart-contract-best-practices/development-recommendations/general/external-calls/#dont-use-transfer-or-send

`transfer` and `send` calls are no longer considered best practice
Having said that, I do not have a strong opinion I can change it back to `transfer`

Reply:
Ok lets go with consensys on this. can you remove the gas comment and change the inline comment (this forwards all available gas correct?